db/migrate/20170421153056_add_email_count_to_signatures.rb

```diff
@@ -0,0 +1,5 @@
+class AddEmailCountToSignatures < ActiveRecord::Migration
+  def change
+    add_column :signatures, :email_count, :integer
```

Comment:
@AbraaoMota on balance I think yes, since we won't merge it until we've archived the petitions and signatures.